scss/mixins/_star-rating.scss

```diff
@@ -1,5 +1,9 @@
 // scss-docs-start form-star-rating-mixin
-@mixin form-star-rating($color) {
-  background-image: escape-svg(url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 25 25'><path fill='#{$color}' d='m12.5 2.9 2.379 7.35 7.721-.007-6.25 4.536 2.392 7.346-6.242-4.547-6.242 4.547 2.392-7.346-6.25-4.536 7.721.007Z'/></svg>"));
+@mixin form-star-rating($color, $background-color) {
```

Comment:
Maybe rename `$color` to `$border-color` for better understanding? Same below

Reply:
I gave it a second thought and I came with the idea that `$path-color` and `$fill-color` could be better than `$outer-color` and `$inner-color`. I think it's more explicit and easier to understand.

Any opinion @hannahiss?

Reply:
I realize that I did not answer. One or another seems ok, it's better than `$color`/`$background-color` :-)